### ISSUE TITLE
Clarification on installing toc_with_conda, using conda

### ISSUE THREAD

**Issue by ReddyKilowatt** [fauskanger/tox_with_conda#2]
This page: [https://github.com/fauskanger/tox_with_conda](url) shows how to install tox_with_conda, using conda as follows: 

> conda install -c fauskanger tox_with_conda

However, the user **must** specify which conda environment to install tox_with_conda **in**, using this command, to avoid having it installed in the base (root) conda.

> conda install **-n Python27Env** -c fauskanger tox_with_conda

**Comment by fauskanger** [comment]
Yes that is correct, but I hardly ever use the root environment. In my case, I would first activate my environment for my current project and then do `conda install -c fauskanger tox_with_conda`.

Using the `-n myenv` is an other way of achieving this without having to activate the myenv environment first.

**Comment by fauskanger** [comment]
Thanks for the feedback though, I didn't think about the issue. I'll update the README later.

**Comment by fauskanger** [comment]
On a side-note, I might add that there is no drawbacks by using `pip install tox_with_conda` AFAIK. I just added a conda package as it felt wrong to have a conda-utility that wasn't on conda 😃 )

**Comment by ReddyKilowatt** [comment]
> In my case, I would first activate my environment for my current project and then do

Activating the environment doesn't make "conda install" install it in that environment.
That's always been a pet peeve of mine. This is why I suggested the conda -n "envname" install

**Comment by ReddyKilowatt** [comment]
> Using the `-n myenv` is an other way of achieving this without having to activate the myenv environment first.

That's _exactly_ what my suggestion is.

It looks like conda installs packages in the currently activate environment, but it wasn't always like that.

**Comment by fauskanger** [comment]
> That's _exactly_ what my suggestion is.

Yes, I only wanted to acknowledge your suggestion as a viable alternative to install from an active env. 

> It looks like conda installs packages in the currently activate environment, but it wasn't always like that.

Thanks for bringing this to my attention. I have used conda only for a few years so I wasn't aware that this was a concern before.
